pkg/api/converterfromv20180930preview.go

```diff
@@ -45,21 +47,22 @@ func ConvertFromV20180930preview(oc *v20180930preview.OpenShiftManagedCluster, o
 	return cs, nil
 }
 
-func mergeResourcePurchasePlan(oc *v20180930preview.OpenShiftManagedCluster, cs *OpenShiftManagedCluster) {
-	if oc.Plan == nil {
-		return
+// convertFromResourcePurchasePlanv20180930preview merges filled out fields from the external API to the internal representation, doesn't change nil fields.
+func convertFromResourcePurchasePlanv20180930preview(oc *v20180930preview.OpenShiftManagedCluster, cs *OpenShiftManagedCluster) {
```

Comment:
This is still merging and not just conversion. Any reason why we would merge here but passthrough via the admin api?

Reply:
/cc @jim-minter